lms/templates/credit_notifications/credit_eligibility_email.html

```diff
@@ -34,7 +34,17 @@
             </p>
 
             <p>
-              ${_(u"Congratulations! You are eligible to receive course credit for successfully completing your {platform_name} course! {link_start}Get your credit now.{link_end}").format(
+               ${_(u"Congratulations! You are eligible to receive course credit from ")}
+                % if providers:
```

Comment:
@marcotuts Please guide how to show the multiple providers. simple comma separated list is fine or some thing else ?